### PR TITLE
Creating Token

### PR DESCRIPTION
**Pull request by sureshn04**
Creating a Token based on Register time.

### CODE REVIEW

File: totp.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const crypto = require("crypto");
+
+const db = {
+  username: "oslevvce",
+  password: "123osl",
+  secrete: 1561115594546
+};
```
Comment (madhurish) [inline totp.js:7]: See if there is a better way to store DB credentials than saving the sting in an opensource repo. Also is your DB local? I'd recommend not sharing the credentials out in the open, take care of spellings: oslevvce -> oslvvce. secrete -> secret.

**Review by madhurish — CHANGES_REQUESTED** [review]

**Comment by shreevari** [comment]
You can add the DB connection and fetching the Registration Time(RT) and password in the next commit I guess.

**Comment by suresh-44** [comment]
Sorry for the spelling mistake, yeah that credentials only for the testing purpose. And also I'm no doing anything with the that object.

File: totp.js (hunk 1 above)
Comment (sureshn04) [reply totp.js:7]: Sorry for the spelling mistake, yeah that credentials only for the testing purpose. And also I'm no doing anything with the that object.

**Pull request merged**